frontend/src/App.tsx

```diff
@@ -16,18 +26,19 @@ class App extends PureComponent<Props> {
 
   render() {
     return (
-      <>
-        <ImageContainer
-          ref={(instance) => {
-            this.imageContainer = instance;
-          }}
-        />
-        <ImageUploadContainer
-          onUploadedCallback={() => {
-            this.imageContainer?.fetchImages();
-          }}
-        />
-      </>
+      <Layout style={{ height: '100%' }}>
+        <Header style={{ color: 'white' }}>
+          <Logo>POV Display Control Panel</Logo>
+        </Header>
+        <Layout>
+          <Content style={{ padding: '32px 48px 0' }}>
+            <ImageContainer ref={(instance) => (this.imageContainer = instance)} />
+          </Content>
+          <Sider style={{ color: 'white' }}>
+            <ImageUploadContainer onUploadedCallback={() => this.imageContainer?.fetchImages()} />
```

Comment:
_Question_: As a naive `React` user here, I probably would've implemented new image uploaded state-propogation/re-rendering by editing the `App.props` instead of via a callback. What are the pros/cons of doing this via callbacks instead of `props`? 

My intuition is that re-fetching entirely is _better_ in that after a user uploads then any concurrent changes made by other users will be picked up. However that does mean that there is an extra network RTT for doing the `GET api/images` + `GET api/images/{uploadedId}` (I believe a props change would only re-trigger a fetch for `GET api/images/{uploadedId}` since all other images are probably already cached depending on the browser + server combo). 

_Note:_ This is really just a question for my education on React best practices 😄